test/dependencies.jl

```diff
@@ -122,11 +122,11 @@ end
             dependencies = [
                 Dependency("Zlib_jll")
             ]
-            platform = HostPlatform()
+            platform = HostPlatform(parse(Platform, join(["x86_64", split(Base.BinaryPlatforms.host_triplet(), "-")[2:end]...], "-")))
             ap = @test_logs setup_dependencies(prefix, getpkg.(dependencies), platform)
             @test "libz." * platform_dlext(platform) in readdir(last(libdirs(Prefix(destdir(dir, platform)))))
             @test "zlib.h" in readdir(joinpath(destdir(dir, platform), "include"))
-            @test readdir(joinpath(destdir(dir, platform), "logs")) == ["Zlib.log.gz"]
+            @test "Zlib.log.gz" in readdir(joinpath(destdir(dir, platform), "logs"))
```

Comment:
Why?  This isn't equivalent and the test that was before was meant to test that in that directory there is _only_ that file.

Reply:
I'll need to dig deeper tomorrow then, not sure why other files are popping up:

```Julia
Evaluated: ["Zlib.log.gz", "fix_identity_mismatch_libz.1.2.11.dylib.log.gz", "ldid_libz.1.2.11.dylib.log.gz"] == ["Zlib.log.gz"]
```

Reply:
OK, so it looks like the Mac builds just have more log files in the zlib binary.